pkg/cloudevents/transport/http/transport.go

```diff
@@ -353,6 +353,12 @@ func (t *Transport) StartReceiver(ctx context.Context) error {
 	}
 }
 
+// HasTracePropagation implements Transport.HasTracePropagation
+// TODO: implement HTTP trace propagation.
```

Comment:
this todo means there is still more to enable this feature?

Reply:
I've gone ahead and added HTTP propagation in this PR.